website/content/sdk/quickstart/installation.md

```diff
@@ -1,7 +1,4 @@
----
-title: Installation
-sidebar_position: 2
----
+# Installation
```

Comment:
We make here an exception to the format to allow to look nicely on https://pypi.org/project/openbb/.